modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -11,36 +11,15 @@ The *Pipelines* view in the *Developer* perspective lists all the Pipelines in a
 .Procedure
 . In the *Pipelines* view of the *Developer* perspective, select a project from the *Project* drop-down list to see the Pipelines in that project.
 +
-//image::op-pipeline-list.png[Pipelines in the Developer perspective]
+image::op-pipeline-list.png[Pipelines in the Developer perspective]
```

Comment:
The admin side changed our filters for the list pages... just FYI, it looks differently.

Reply:
Updated this and another image and debugged the couple of images that did not render earlier.